dataset-registry/generate.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# Setup script env
+
+# e Exit immediately if a command exits with a non-zero exit status.
+# u Treat unset variables as an error when substituting.
+# x Print commands and their arguments as they are executed.
+set -eux
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="dataset-registry"
+REPO_PATH="$HERE/build/$REPO_NAME"
+
+if [ -d "$REPO_PATH" ]; then
+    echo "Repo $REPO_PATH already exists, remove it first."
+    exit 1
+fi
+
+mkdir -p $REPO_PATH
+pushd $REPO_PATH
+
+# Create virtualenv, install `dvc`, initialize/config DVC project
+
+virtualenv -p python3 .env
+export VIRTUAL_ENV_DISABLE_PROMPT=true
+source .env/bin/activate
+echo '.env/' >> .gitignore
+
+pip install dvc[s3]
+
+git init
+dvc init
+
+# Remote active on this environment only for writing to HTTP redirect below.
+dvc remote add -d --local storage s3://dvc-public/remote/dataset-registry
+
+# Actual remote for generated project (read-only). Redirect of S3 bucket below.
+dvc remote add -d storage https://remote.dvc.org/dataset-registry
+
+cp $HERE/code/README.md $REPO_PATH
+
+git add .
+git commit -m "Init & config DVC project, add README"
+
+# Get Started
+
+mkdir get-started
+wget https://data.dvc.org/get-started/data.xml -O get-started/data.xml
+dvc add get-started/data.xml
```

Comment:
Question @shcheklein: Shouldn't data.xml be a zip file on our S3 bucket and dataset-registry repo remote? Like in https://dvc.org/doc/get-started/example-versioning

Reply:
that was a conscious decision to use the plain `data.xml` in get started. The reason - simplicity (to avoid introducing an extra step with unzip). It was beneficial to include it in the tutorial to show that DVC is language agnostic, basically to show that it can run any command, not only python. 

So, I would keep it XML for the get started. We can decide later when we start consolidating tutorial and NLP example if we want to keep ZIP.